web-marketplace/src/components/organisms/ChooseCreditsForm/ChooseCreditsForm.tsx

```diff
@@ -163,6 +168,19 @@ export function ChooseCreditsForm({
     name: CURRENCY_AMOUNT,
   });
 
+  const { isLoading, userBalance: microUserBalance } = useFetchUserBalance(
```

Comment:
this breaks https://deploy-preview-2501--regen-storybook.netlify.app/?path=/story/marketplace-organisms-choosecreditsform--choose-credits
maybe we could just pass it as a prop?

Reply:
we might need this logic for updating the credits amount from the order summary card so it would make sense to move it one level up

@erikalogie what should we display in case the user enters a amount of credits from the order summary card that results in an amount to pay that exceeds user balance?

Reply:
See this flow starting here: https://www.figma.com/design/BTuUv6QXY4GbliZcXCe8RJ/Fiat-payments?node-id=5314-94485&t=tyf73iuZkQA0S9cW-1

Reply:
@r41ph @blushi I updated the flow for when it exceeds the user's balance: https://www.figma.com/design/BTuUv6QXY4GbliZcXCe8RJ/Fiat-payments?node-id=5843-92226&t=tyf73iuZkQA0S9cW-1